x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/Watcher.java

```diff
@@ -613,4 +624,21 @@ static void validAutoCreateIndex(Settings settings, Logger logger) {
     public void close() throws IOException {
         IOUtils.closeWhileHandlingException(httpClient);
     }
+
+
+    /**
+     * Reloads all reloadable services' settings.
+     * @param settings
+     *            Settings used while reloading the plugin. All values are
+     *            retrievable, including the values stored in the node's keystore.
+     *            The setting values are the initial ones, from when the node has be
+     *            started, i.e. they don't follow dynamic updates.
+     */
```

Comment:
this is already documented in `ReloadablePlugin` I suppose?

Reply:
Yea intellij automagically placed it here... i assumed that was a good thing (tm) but ill gladly remove it.